src/cloud/iotile-cloud-serv.ts

```diff
@@ -294,9 +294,9 @@ export class IOTileCloud {
    *
    * Returns an array of OrgMetaData objects with all organizations retrieved.
    */
-  public async fetchOrgMetaData() : Promise<OrgMetaData[]>{
+  public async fetchOrgMetaData(filter?: ApiFilter) : Promise<OrgMetaData[]>{
```

Comment:
It is confusing to have a filter on a function to fetch Orgs that is not filtering the Orgs themselves but filtering projects. At the minimum, the argument should be called `projectFilter` or something so you can potentially add an orgFilter in the future as well.

Reply:
Agreed, will update

Reply:
Do we want to go ahead and add optional filters for orgs as well?